### PR TITLE
Deprecate memcached

### PR DESCRIPTION
**Pull request by TimWolla**

### CODE REVIEW

**Review by dtdesign — COMMENTED** [review]
I would suggest displaying it as `Memcached (Not Recommended)` as it is far stronger than a simple word on deprecation. Pointing out that the Memcache support is deprecated within the description is perfectly fine though.

File: wcfsetup/install/lang/en.xml [hunk 1]
```diff
@@ -1233,19 +1233,19 @@ ATTENTION: The messages listed above are greatly shortened. You can view details
...
-		<item name="wcf.acp.option.cache_source_memcached_host.description"><![CDATA[You can specify a weight factor to allow load balancing, for example “localhost:11211:67” or “10.0.13.37:31337:33”. Enter one server per line.]]></item>
+		<item name="wcf.acp.option.cache_source_memcached_host.description"/>
 		<item name="wcf.acp.option.cache_source_redis_host"><![CDATA[Redis-Server]]></item>
 		<item name="wcf.acp.option.cache_source_redis_host.description"><![CDATA[The server’s address, for example “localhost” or “10.0.13.37:1337”.]]></item>
 		<item name="wcf.acp.option.cache_source_type"><![CDATA[Caching Method]]></item>
 		<item name="wcf.acp.option.cache_source_type.description"><![CDATA[Caching methods different from filesystem require special extensions or services running on your machine.]]></item>
 		<item name="wcf.acp.option.cache_source_type.disk"><![CDATA[Use Filesystem (default)]]></item>
-		<item name="wcf.acp.option.cache_source_type.memcached"><![CDATA[Use Memcached]]></item>
+		<item name="wcf.acp.option.cache_source_type.memcached"><![CDATA[Use Memcached (Deprecated)]]></item>
```
Comment (TimWolla) [inline wcfsetup/install/lang/en.xml:1242]:
```suggestion
		<item name="wcf.acp.option.cache_source_type.memcached"><![CDATA[Use Memcached (Not Recommended)]]></item>
```

File: wcfsetup/install/lang/en.xml [hunk 2]
```diff
@@ -1233,19 +1233,19 @@ ATTENTION: The messages listed above are greatly shortened. You can view details
...
 		<item name="wcf.acp.option.cache_source_type.redis"><![CDATA[Use Redis]]></item>
 		<item name="wcf.acp.option.category.general"><![CDATA[General]]></item>
 		<item name="wcf.acp.option.category.general.cache"><![CDATA[Cache]]></item>
 		<item name="wcf.acp.option.category.general.cache.general"><![CDATA[General]]></item>
-		<item name="wcf.acp.option.category.general.cache.memcached"><![CDATA[Memcached]]></item>
-		<item name="wcf.acp.option.category.general.cache.memcached.description"><![CDATA[Memcached uses the machine’s memory to store frequently accessed data, reducing database and filesystem load. Read more about Memcached on <a href="https://memcached.org/" class="externalURL">memcached.org</a>.]]></item>
+		<item name="wcf.acp.option.category.general.cache.memcached"><![CDATA[Memcached (Deprecated)]]></item>
```
Comment (TimWolla) [inline wcfsetup/install/lang/en.xml:1247]:
```suggestion
		<item name="wcf.acp.option.category.general.cache.memcached"><![CDATA[Memcached (Not Recommended)]]></item>
```

File: wcfsetup/install/lang/en.xml [hunk 3]
```diff
@@ -260,7 +260,7 @@
 		<item name="wcf.acp.cache.list.perm"><![CDATA[Permissions]]></item>
 		<item name="wcf.acp.cache.list.size"><![CDATA[Size]]></item>
 		<item name="wcf.acp.cache.source.type.DiskCacheSource"><![CDATA[Filesystem]]></item>
-		<item name="wcf.acp.cache.source.type.MemcachedCacheSource"><![CDATA[Memcached]]></item>
+		<item name="wcf.acp.cache.source.type.MemcachedCacheSource"><![CDATA[Memcached (Deprecated)]]></item>
```
Comment (TimWolla) [inline wcfsetup/install/lang/en.xml:263]:
```suggestion
		<item name="wcf.acp.cache.source.type.MemcachedCacheSource"><![CDATA[Memcached (Not Recommended)]]></item>
```

File: wcfsetup/install/lang/de.xml [hunk 4]
```diff
@@ -1256,19 +1256,19 @@ ACHTUNG: Die oben genannten Meldungen sind stark gekürzt. Sie können Details z
...
-		<item name="wcf.acp.option.cache_source_type.memcached"><![CDATA[Memcached]]></item>
+		<item name="wcf.acp.option.cache_source_type.memcached"><![CDATA[Memcached (Veraltet)]]></item>
 		<item name="wcf.acp.option.cache_source_type.redis"><![CDATA[Redis]]></item>
 		<item name="wcf.acp.option.category.general"><![CDATA[Allgemein]]></item>
 		<item name="wcf.acp.option.category.general.cache"><![CDATA[Cache]]></item>
 		<item name="wcf.acp.option.category.general.cache.general"><![CDATA[Allgemein]]></item>
-		<item name="wcf.acp.option.category.general.cache.memcached"><![CDATA[Memcached]]></item>
-		<item name="wcf.acp.option.category.general.cache.memcached.description"><![CDATA[Memcached speichert häufig benötigte Daten im Arbeitsspeicher zwischen. Dies kann die Last auf die Datenbank und das Dateisystem drastisch reduzieren. {if LANGUAGE_USE_INFORMAL_VARIANT}Lies{else}Lesen Sie{/if} mehr über dieses Thema auf der folgenden Seite: <a href="https://memcached.org/" class="externalURL">memcached.org</a>.]]></item>
+		<item name="wcf.acp.option.category.general.cache.memcached"><![CDATA[Memcached (Veraltet)]]></item>
```
Comment (TimWolla) [inline wcfsetup/install/lang/de.xml:1270]:
```suggestion
		<item name="wcf.acp.option.category.general.cache.memcached"><![CDATA[Memcached (Obsolet)]]></item>
```

File: wcfsetup/install/lang/de.xml [hunk 5]
```diff
@@ -1256,19 +1256,19 @@ ACHTUNG: Die oben genannten Meldungen sind stark gekürzt. Sie können Details z
...
+		<item name="wcf.acp.option.cache_source_memcached_host.description"/>
 		<item name="wcf.acp.option.cache_source_redis_host"><![CDATA[Redis-Server]]></item>
 		<item name="wcf.acp.option.cache_source_redis_host.description"><![CDATA[Die Adresse des Servers, zum Beispiel „localhost“ oder „10.0.13.37:1337“.]]></item>
 		<item name="wcf.acp.option.cache_source_type"><![CDATA[Cache-Methode]]></item>
 		<item name="wcf.acp.option.cache_source_type.description"><![CDATA[{if LANGUAGE_USE_INFORMAL_VARIANT}Beachte{else}Beachten Sie{/if}, dass einige Methoden spezielle Anforderungen an das Server-System stellen und nicht auf jedem Server zur Verfügung stehen.]]></item>
 		<item name="wcf.acp.option.cache_source_type.disk"><![CDATA[Dateisystem (Standard)]]></item>
-		<item name="wcf.acp.option.cache_source_type.memcached"><![CDATA[Memcached]]></item>
+		<item name="wcf.acp.option.cache_source_type.memcached"><![CDATA[Memcached (Veraltet)]]></item>
```
Comment (TimWolla) [inline wcfsetup/install/lang/de.xml:1265]:
```suggestion
		<item name="wcf.acp.option.cache_source_type.memcached"><![CDATA[Memcached (Obsolet)]]></item>
```

**Review by dtdesign — APPROVED** [review]

**Pull request merged**